Uplift/Views/GymDetail/Facilities/MiscellaneousInfoView.swift

```diff
@@ -8,29 +8,32 @@
 
 import UIKit
 
-class MiscellaneousInfoView: UIView {
+class MiscellaneousInfoView: UICollectionViewCell {
 
     private let textView = UITextView()
 
-    init(miscellaneousInfo: [String]) {
+    init() {
         super.init(frame: CGRect.zero)
 
-        let display = formatMiscellaneous(miscellaneousInfo)
-        textView.attributedText = generateAttributedString(for: display, font: UIFont._14MontserratRegular!)
         textView.isEditable = false
         textView.isSelectable = false
         textView.isScrollEnabled = false
         textView.textContainerInset = .zero
         textView.textContainer.lineFragmentPadding = 0
 
-        addSubview(textView)
+        contentView.addSubview(textView)
         setupConstraints()
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
+    func configure(for miscellaneousInfo: [String]) {
+        let display = MiscellaneousInfoView.formatMiscellaneous(miscellaneousInfo)
+        textView.attributedText = MiscellaneousInfoView.generateAttributedString(for: display, font: UIFont._14MontserratRegular!)
+    }
+
     func setupConstraints() {
```

Comment:
make private

Reply:
actually since its only one constraint maybe we could put this in the `init` and remove `setupConstraints`